mage_ai/shared/security.py

```diff
@@ -10,15 +11,33 @@
 ])
 
 
-def filter_out_env_var_values(value: str):
+def filter_out_env_var_values(value: str) -> str:
     env_var_values = dict(os.environ).values()
     whitelisted_env_var_values = set([os.getenv(k) for k in WHITELISTED_ENV_VARS if os.getenv(k)])
     env_var_values = [v for v in env_var_values
-                      if v and len(v) >= MIN_SECRET_ENV_VAR_LENGTH
+                      if v and len(v) >= MIN_SECRET_LENGTH
                       and v not in whitelisted_env_var_values]
-    env_var_values.sort(key=len, reverse=True)
-    value_clean = value
-    for env_var_value in env_var_values:
-        replace_value = '*' * len(env_var_value)
-        value_clean = value_clean.replace(env_var_value, replace_value)
-    return value_clean
+    return filter_out_values(value, env_var_values)
+
+
+def filter_out_config_values(log: str, config: Dict) -> str:
+    """
```

Comment:
For the method comment, let's use the standard format of the docstring.
Example: https://github.com/mage-ai/mage-ai/blob/master/mage_ai/io/base.py#L56-L67